### PR TITLE
Upgrade leak_tracker.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/flutter_test/test/utils/leaking_classes.dart [hunk 1]
```diff
@@ -1,54 +0,0 @@
-// Copyright 2014 The Flutter Authors. All rights reserved.
```
Comment (christopherfujino) [inline packages/flutter_test/test/utils/leaking_classes.dart:1]: I'm guessing this logic was upstreamed to leak_tracker?

**Review by christopherfujino — APPROVED** [review]
LGTM

File: packages/flutter_test/test/utils/leaking_classes.dart (hunk 1 above)
Comment (polina-c) [reply packages/flutter_test/test/utils/leaking_classes.dart:1]: yes!

Comment (christopherfujino) [reply packages/flutter_test/test/utils/leaking_classes.dart:1]: nice

**Pull request merged**